src/Interactive/EditorFeatures/Core/Completion/CompletionProviders/ReplCommandCompletionProvider.cs

```diff
@@ -0,0 +1,63 @@
+﻿// Copyright (c) Microsoft.  All Rights Reserved.  Licensed under the Apache License, Version 2.0.  See License.txt in the project root for license information.
+
+using System.Text.RegularExpressions;
+using System.Threading;
+using System.Threading.Tasks;
+using Microsoft.CodeAnalysis.Completion;
+using Microsoft.CodeAnalysis.Editor.Implementation.Interactive;
+using Microsoft.CodeAnalysis.Shared.Extensions;
+using Microsoft.CodeAnalysis.Text;
+using Microsoft.VisualStudio.InteractiveWindow;
+using Microsoft.VisualStudio.Text.Editor;
+using Microsoft.VisualStudio.Utilities;
+using Microsoft.VisualStudio.InteractiveWindow.Commands;
+using Roslyn.Utilities;
+
+namespace Microsoft.CodeAnalysis.Editor.Completion.CompletionProviders
+{
+    [ExportCompletionProvider("ReplCommandCompletionProvider", LanguageNames.CSharp)]
```

Comment:
Is this supposed to be here?  I would have thought it only went on concrete types.  If it does belong here, VB should probably be mentioned as well.